emily_landi/.gitignore

```diff
@@ -0,0 +1,2 @@
+**/node_modules
+**/.DS_Store
```

Comment:
Your log folder should also be ignored. Usually there's no need to have app-created files on your repo.